pjsip/src/pjsua-lib/pjsua_vid.c

```diff
@@ -1868,6 +1882,13 @@ static pj_status_t call_reoffer_sdp(pjsua_call_id call_id,
 	return PJSIP_ESESSIONSTATE;
     }
 
+    /* Notify application */
+    if (!call->hanging_up && pjsua_var.ua_cfg.cb.on_call_sdp_created) {
+	(*pjsua_var.ua_cfg.cb.on_call_sdp_created)(call_id, sdp,
+						   call->inv->pool_prov,
+						   NULL);
+    }
+
```

Comment:
This does not look related to the codec? If it is a fix for another issue, please put it in another PR or misc PR.